simulator/mips/mips_instr.h

```diff
@@ -98,6 +100,7 @@ class FuncInstr
             OUT_I_STORER,
             OUT_J_JUMP,
             OUT_J_JUMP_LINK,
+            OUT_RI_JUMP_LINK,
```

Comment:
Could you please rename it to "OUT_RI_BRANCH_LINK"? MIPS uses "jump" name for unconditional jumps (both direct and indirect).